ArcPicker/ArcPicker.js

```diff
@@ -0,0 +1,244 @@
+		 *
+		 * @type {Node}
+		 * @public
+		 */
+		slotCenter: PropTypes.node,
+
+		/**
+		 * The start angle(in degrees) of the arc.
+		 *
+		 * The value should be between 0 and 360.
+		 *
+		 * @type {Number}
+		 * @default 50
+		 * @public
+		 */
+		startAngle: PropTypes.number,
+
+		/**
+		 * The stroke width of the arc picker.
+		 *
+		 * @type {Number}
+		 * @default 6
+		 * @public
+		 */
+		strokeWidth: PropTypes.number,
+
+		/**
+		 * Value of ArcPicker.
+		 *
+		 * @type {Number|String}
+		 * @public
+		 */
+		value: PropTypes.oneOfType([PropTypes.number, PropTypes.string])
+	},
+
+	defaultProps: {
+		backgroundColor: '#eeeeee',
+		endAngle: 310,
+		foregroundColor: '#444444',
+		radius: 150,
+		startAngle: 50,
+		strokeWidth: 6
+	},
+
+	styles: {
+		css,
+		className: 'arcPicker'
+	},
+
+	computed: {
+		arcSegments: (props) => {
+			const {backgroundColor, children, endAngle, foregroundColor, onClick, radius, selectionType, startAngle, strokeWidth, value} = props;
+
+			return (
+				children.map((option, index) => {
+					// Calc `arcStartAngle`, `arcEndAngle` based on `startAngle` and `endAngle` for every <Arc />
+					const pauseAngle = 2;
+					const arcSegments = children.length;
+					const arcStartAngle = startAngle + (endAngle - startAngle) / arcSegments * index;
+					const arcEndAngle = startAngle + (endAngle - startAngle) / arcSegments * (index + 1) - pauseAngle;
+
+					const color = (selectionType === 'cumulative' && value > option || value === option) ? foregroundColor : backgroundColor;
+
+					return (
+						<Arc
+							className={css.arc}
+							color={color}
+							endAngle={arcEndAngle}
+							key={index}
+							onClick={onClick(option)}
+							radius={radius}
+							startAngle={arcStartAngle}
+							strokeWidth={strokeWidth}
+						/>
+					);
+				}));
+		},
+		style: ({radius, style}) => {
+			const size = ri.scaleToRem(radius * 2);
+			return {...style, height: size, width: size};
+		}
+	},
+
+	render: ({arcSegments, slotCenter, ...rest}) => {
+		delete rest.backgroundColor;
+		delete rest.children;
+		delete rest.endAngle;
+		delete rest.foregroundColor;
+		delete rest.onClick;
+		delete rest.selectionType;
+		delete rest.startAngle;
+		delete rest.value;
+
+		return (
+			<div {...rest} className={css.arcPicker}>
+				{arcSegments}
+				<div className={css.valueDisplay}>
+					{slotCenter}
+				</div>
+			</div>
+		);
+	}
+});
+
+/**
+ * Applies Agate specific behaviors to [ArcPicker]{@link agate/ArcPicker.ArcPicker} components.
+ *
+ * @hoc
+ * @memberof agate/ArcPicker
+ * @mixes agate/ArcPicker.ArcPickerBehaviorDecorator
+ * @mixes agate/Skinnable.Skinnable
+ * @public
+ */
+const ArcPickerDecorator = compose(
+	Changeable,
+	ArcPickerBehaviorDecorator,
+	Skinnable
+);
+
+/**
+ * An arc picker component, ready to use in Agate applications.
+ *
+ * Usage:
+ * ```
+ * <ArcPicker backgroundColor="#444444" endAngle={200} foregroundColor="#eeeeee" selectionType="single" startAngle={0} />
```

Comment:
Please remove `backgroundColor` and `foregroundColor` here if it's not required.

Reply:
Same above.